udata-tabular-preview/explore.vue

```diff
@@ -58,29 +74,67 @@ export default defineComponent({
     const columnCount = ref(null);
     const loading = ref(true);
     const hasError = ref(false);
-    configure({ csvapiUrl: tabular_csvapi_url, pageSize: tabular_page_size });
-    apify(props.resource.url).then(res => {
+
+    /** @type {import("vue").Ref<string | null>} */
+    const sortBy = ref(null);
+    const sortDesc = ref(false);
+
+    /**
+     *
+     * @param {string} col
+     */
+    const sortbyfield = (col) => {
+      sortDesc.value = !sortDesc.value
+      if(sortBy.value !== col) {
+        sortDesc.value = false
+      }
```

Comment:
I would find it more explicit like this
```suggestion
      if(sortBy.value == col) {
        sortDesc.value = !sortDesc.value
      } else {
        sortDesc.value = false
      }
```